### DISCUSSION TITLE
Any reason behind the subsetting of MetNO data?

### DISCUSSION THREAD

**Discussion by guidocioni** [open-meteo/open-meteo#596] [Q&A]
The original MEPS dataset (https://thredds.met.no/thredds/metno.html) has many variables available, and also ensemble realizations for some of them.
However, in open-meteo only a really small subset of these variables are present, and no ensemble data is implemented.
Is there any particular reason behind this?

**Comment by patrick-zippenfenig** [comment]
There is no particular reason why it is not available. At the time I implemented it, I did not integrate any ensemble data. I am also only using the post-processed version with fewer variables. Once issue is that only 2.5 days of forecast are available and there is no good fit to combine it with a global model. ECMWF IFS would be ideal, but the open-data distribution is limited to 0.4° and 3-hourly time-steps.

**Reply by guidocioni** [reply]
Mmmm ok. I think it would be good to implement ensemble data at least for a few interesting variables (like precipitation, temperature and snow depth) to get higher accuracy of winter conditions over Scandinavia. 

For the deterministic part (forecast endpoint) maybe there is no strong need to change: I only discovered this because when doing the vertical plot I realized that MetNO had no data on pressure levels, although this was present in the remote thredds catalogue, so I was kind of confused.
